kong/templates/nginx_kong.lua

```diff
@@ -34,6 +34,7 @@ lua_shared_dict cache ${{MEM_CACHE_SIZE}};
 lua_shared_dict reports_locks 100k;
 lua_shared_dict cluster_locks 100k;
 lua_shared_dict cluster_autojoin_locks 100k;
+lua_shared_dict cache_locks 100k;
```

Comment:
we do not need so many different `*_locks` shared dicts. A single one is able to hold multiple locks with different options (expiration, timeout, etc...). It will be worth cleaning this up some time soon.